scripts/src/main/resources/scripts/command/ide

```diff
@@ -448,6 +450,33 @@ function doUninstall() {
   doSuccess "Goodbye and have a nice day!"
 }
 
+function doCreate() {
+    local create_path="${1}"
+    if [ -z "${create_path}" ]
+    then
+      create_path="$(pwd)"
+    fi
+
+    if [ ! -d "${create_path}" ]
+    then
+      mkdir -p ${create_path}
+    fi
+
+    count_lines="$(ls -alF ${create_path} | wc -l)"
+    if [ ${count_lines} -gt 3 ]
+    then
+      doAskToContinue "Directory is not empty, continue?"
+    fi
+  
+    SETTINGS_PATH="${create_path}/settings"
+    doUpdateSettings
+    
+    doUpgradeMavenArtifact "${create_path}/" "${DEVON_IDE_REPO_URL}" "devonfw-ide-scripts" "${target_version}" ".tar.gz" 
+
+    cd ${create_path}
+    ${create_path}/setup
```

Comment:
1. please pass the arguments to the settings: first call `shift` then pass `${@}`.
2. On windows we usually run the `setup.bat` that does some extra things (registry tweaks for explorer context menu). However, as we can only run the command `devon ide create ...` if devonfw-ide is already installed, we can assume that these extra things have already been done. So that should be fine and we do not need to do anything for 2. if it works on windows and all other platforms.

Reply:
2. I compared setup with setup.bat and found the differences for Windows. But if we start devon ide create, there is already the additional things for windows done. Ant I think we don't need to run them twice.

Reply:
1. done